addons/policies/bootstrap_entity_policies.json

```diff
@@ -3075,6 +3075,357 @@
                     "entity-update"
                 ]
             }
+        },
```

Comment:
We can club following together:
* CUD -> BusinessPolicy + BusinessPolicyException + BusinessPolicyIncidents + Incident +  BusinessPolicyLog + ApprovalWorkflowRequest 
* READ  -> BusinessPolicy + BusinessPolicyException + BusinessPolicyIncidents + Incident

As these policies have same resources entity-classification & entity & also same policyRoles